SixLabors.Fonts.sln

```diff
@@ -38,8 +38,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "samples", "samples", "{71A3
 		samples\Fonts.Samples.ruleset = samples\Fonts.Samples.ruleset
 	EndProjectSection
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DrawWithImageSharp", "samples\DrawWithImageSharp\DrawWithImageSharp.csproj", "{999EDFB3-9FE4-4E09-B669-CB02E597EC20}"
```

Comment:
This is temporary due to breaking changes re `IGlyphRenderer`

Reply:
I have a local version of https://github.com/SixLabors/ImageSharp.Drawing/pull/211 that works with these changes.